server/src/main/java/org/opensearch/search/aggregations/metrics/MinAggregator.java

```diff
@@ -104,6 +104,22 @@ public ScoreMode scoreMode() {
         return valuesSource != null && valuesSource.needsScores() ? ScoreMode.COMPLETE : ScoreMode.COMPLETE_NO_SCORES;
     }
 
+    @Override
```

Comment:
nit : This can be moved up to maybe `NumericMetricsAggregator.SingleValue` or some other common place since all the four are same implementations.

Reply:
Yeah -- unfortunately, we don't (yet) have a good base class, since `NumericMetricsAggregator.SingleValue` is also the parent of several aggregators that don't have star tree metrics, like `WeightedAvgAggregator` and `MedianAbsoluteDeviationAggregator`.

I think we might need something between `NumericMetricsAggregator.SingleValue` and the various `MetricStats` aggregators to hold this common logic.